jobsapp/models.py

```diff
@@ -9,6 +11,19 @@
 from notifications.decorators import event_dispatcher
 from notifications.events import EVENT_NEW_JOB
 
+# Global Imports
+JOB_INDEXES = ("location", "category", "type", "title")
+
+# Job remote types
+NO_REMOTE = "1"
```

Comment:
It's better to move this options inside the model. For example:

```python
class Job(models.Model):
  NO_REMOTE = "1"
  REMOTE = "2"
  REMOTE_CHOICES = ....
```

It will allow to use this constants thereby:

```
Job.REMOTE
```